Dockerfile

```diff
@@ -0,0 +1,19 @@
+#
+# Build: docker build -t hocr-tools-app .
+# Start: docker run -it --rm -v ${PWD}:/usr/src/app hocr-tools-app bash
+# Test: ./test/tsht
+#
+
+FROM python:2
+
+RUN apt-get update && apt-get install -y pdfgrep
+
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+COPY requirements.txt /usr/src/app/
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY . /usr/src/app
+
+RUN ["python", "setup.py", "install"]
```

Comment:
Was just wondering whether/why you wanted to prevent shell execution for this command but not for the other `RUN` directives. If it's just about form, I'd stick with the plain string version so you can smush all the RUN commands in one to reduce layer size. But it's not important now.